AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/internal/helper/AndroidManifestFinder.java

```diff
@@ -291,13 +293,31 @@ private AndroidManifest parse(File androidManifestFile, boolean libraryProject)
 		componentQualifiedNames.addAll(receiverQualifiedNames);
 		componentQualifiedNames.addAll(providerQualifiedNames);
 
+		Map<String, String> metaDataQualifiedNames = new HashMap<String, String>();
+		NodeList metaDataNodes = documentElement.getElementsByTagName("meta-data");
+		for (int i = 0; i < metaDataNodes.getLength(); i++) {
+			Node node = metaDataNodes.item(i);
+			Node nameAttribute = node.getAttributes().getNamedItem("android:name");
+			Node valueAttribute = node.getAttributes().getNamedItem("android:value");
+
+			if (nameAttribute == null || valueAttribute == null) {
```

Comment:
Checking now the code, I see that also a specific method is called to collect other tasks, I will do the same, to follow the same "rules". This was something I did like 1year ago, so probably the structure of the file was different for then, I will review this too.